src/watts/plugin_sas.py

```diff
@@ -0,0 +1,203 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+from contextlib import redirect_stdout, redirect_stderr
+from datetime import datetime
+from pathlib import Path
+import shutil
+import time
+from typing import List, Optional
+
+import os
+import glob
+import numpy as np
+import pandas as pd
+import platform
```

Comment:
`os`, `glob`, and `platform` belong in the first block of imports. According to PEP8, there should be three blocks: 1) standard-library imports, 2) third-party module imports, and 3) local imports